README.md

```diff
@@ -3,15 +3,30 @@ A starter repo that every TTS internal buy can clone to get started.
 
 ## Instructions
 
-For a project titled `Project X`, run the following to start a fresh repo:
+For a project titled `Project X` and with the repository `tts-buy-project-x` already started, run the following to start a fresh repo:
 
 ```
 git clone https://github.com/18F/tts-internal-buy-starter-repo.git
-mv tts-internal-buy-starter-repo project-x
+mv tts-internal-buy-starter-repo tts-buy-project-x
 cd project-x
 rm -rf .git
 git init
 echo "# Project X" > README.md
 ```
 
 This pulls down the repository from GitHub, renames the folder, restarts the `git` commit history, and removes these instructions.
+
+```
+git add .
+git commit -m "Initial commit"
+git status
```

Comment:
This command isn't required, though it can be helpful to run `git status` frequently to see what's going on.

Reply:
Ah, looks like that's explaining with the following note. Never mind me.